src/resources/communityContributionCompany.ts

```diff
@@ -43,39 +39,20 @@ export const CommunityContributionCompanyResource: ResourceIF = {
         ]
       },
       {
-        header: `Can I use the sample Incorporation Agreement and Articles?`
-      },
-      {
-        header: `You can use the sample Articles if:`,
-        icon: 'mdi-check',
-        iconColor: `green darken-2`,
-        statements: [
-          `There are no special rights or restrictions attached to any class or series of shares in
-            the corporation’s authorized share structure.`,
-          `You do not wish to change any of the standard provisions in the sample Articles.`
-        ]
-      },
-      {
-        header: `You cannot use the sample Articles if:`,
-        icon: 'mdi-close',
-        iconColor: `red`,
-        statements: [
-          `There are special rights or restrictions attached to any class or series of shares in the corporation’s
-            authorized share structure.`,
-          `You wish to change any of the standard provisions in the sample Articles.`
+        header: `Retain the signed Incorporation Agreement and Articles`,
+        helpText: [
+          `The company is required to keep signed copies of the Incorporation Agreement and Articles in the company’s
+          record book. For a complete list of records a company is required to keep please see section 42 of the
+          Business Corporations Act.`
         ]
       }
     ],
+    articleUrl: `Sample_articles.pdf`,
```

Comment:
Is it the article url or article file name?

Reply:
Good point, this is actually the file name. The url to get the file from the public folder is done in the component itself. 
So yea, this is technically just the file name. Will update!

Reply:
Updated!